Leanplum-iOS-SDK-Example/Base.lproj/Main.storyboard

```diff
@@ -100,7 +117,7 @@
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="2fq-zs-ODj" userLabel="First Responder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="2142" y="-1465"/>
+            <point key="canvasLocation" x="2141.5999999999999" y="-1465.8170914542729"/>
```

Comment:
this is prob auto generated, can you anyways make sure that the buttons are nicely centered/aligned :)